features/costs_input.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Costs Input
+  As a Finance guy
```

Comment:
move these features to spec/features and replaced with a simple check of the form data.